### PR TITLE
Cut 0.14.0.

### PR DESCRIPTION
**Pull request by fabxc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
+## 0.14.0 / 2015-06-01
+* [CHANGE] Configuration format changed and switched to YAML.
```
Comment (juliusv) [inline CHANGELOG.md]: Let's add a link to the migration tool here (and synchronize with pushing the new docs).

Comment (juliusv) [inline CHANGELOG.md]: ...and a link to the docs, I wanted to add.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+* [ENHANCEMENT] Simplified binary operation evaluation.
+* [ENHANCEMENT] More stable component initialization.
+* [ENHANCEMENT] Added internal expression testing language.
+* [BUGFIX] Fix graph links with path prefix.
+* [ENHANCEMENT] Allow building from source without git.
+* [ENHANCEMENT] Improve storage iterator performance.
+* [ENHANCEMENT] Change logging output format and flags.
+* [BUGFIX] Fix memory alignment bug for 32bit systems.
+* [ENHANCEMENT] Improve web redirection behavior.
+* [ENHANCEMENT] Allow overriding default hostname for Prometheus-URLs.
```
Comment (juliusv) [inline CHANGELOG.md]: no dash before URLs

**Comment by juliusv** [comment]
:+1: besides nit

**Comment by beorn7** [comment]
Perhaps we should release client_golang first, so that we can set the Godeps to a release tag.
I expect that helps Tincho et al. creating distribution packages.

**Comment by juliusv** [comment]
@beorn7 Hmm, do we know how important that is really? We do want to get 0.14.0 out there, especially in light of the conference talk tomorrow.

**Comment by beorn7** [comment]
It should be a matter of minutes...

Not sure how important it is, because Tincho might just work with commit
id's anyway.

On Mon, Jun 1, 2015 at 6:51 PM, Julius Volz <notifications@github.com>
wrote:

> @beorn7 <https://github.com/beorn7> Hmm, do we know how important that is
> really? We do want to get 0.14.0 out there, especially in light of the
> conference talk tomorrow.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/prometheus/pull/759#issuecomment-107634619>
> .
>



-- 
Björn Rabenstein, Engineer
http://soundcloud.com/brabenstein

SoundCloud Ltd. | Rheinsberger Str. 76/77, 10115 Berlin, Germany
Managing Director: Alexander Ljung | Incorporated in England & Wales with
Company No. 6343600 | Local Branch Office | AG Charlottenburg  | HRB 110657B

**Pull request merged**